packages/blockchain-link/README.md

```diff
@@ -1,55 +1,83 @@
 # @trezor/blockchain-link
 
-API for communication with blockchain backends.
-Every type of a backed is isolated inside own worker file.
-Currently supported:
+blockchain-link is a client and unified interface for several backends (_BE_ further on) of various blockchain networks. Currently, there are implementations for
+- [blockbook](https://github.com/trezor/blockbook): BE developed and deployed by SatoshiLabs. Provides access to Bitcoin(like) and Ethereum(like) networks.
+- [ripple](https://xrpl.org/): third party BE that provides access to the Ripple network.
+
+## API
+
+The API aims to be identical across all backends and networks. However, this isn't entirely possible and there are some discrepancies which are noted below.
+
+All methods can fail and return an error eg. in case the BE is offline.
+
+- `getInfo`: Get general information about the network, like current block-height, name, smallest possible division, etc.
+- `getBlockHash`: Get hash of a block of given height.
+- `getAccountInfo`: Get info about an account, eg. derived addresses, balance, transaction history etc.
+- `getAccountUtxo`: Get unspent inputs for given account. Only bitcoin(like) networks.
+- `getAccountBalanceHistory`: Get historical progression of given account's balance. Used for rendering a graph in Suite's dashboard.
+- `getCurrentFiatRates`: Get current fiat rates.
+- `getFiatRatesForTimestamps`: Get historical fiat rates, only some networks support this.
+- `getFiatRatesTickersList`: Get fiat currencies for which rates are available.
+- `estimateFee`: Get ‘recommended’ fee value inferred from current traffic in the network.
+- `subscribe`: Subscribe for live changes in
+    - blockchain i.e new blocks mined.
+    - accounts, addresses i.e. new transactions broadcasted or mined.
+    - fiatRates
+    - connection to BE (HANDSHAKE, CONNECT, DISCONNECT)
+    
+    Handling subscription state is left to the user. (Hence the purpose of HANDSHAKE, CONNECT and DISCONNECT notifications.)
+- `unsubscribe`: Cancel existing subscription.
+- `getTransaction`: Get info about a given transaction. Return value of this method is not identical across networks; this method exposes the specific transaction format.
```

Comment:
```suggestion
- `getTransaction`: Get info about a given transaction. Return value of this method is not identical across networks; this method exposes raw response from BE.
```

Reply:
edited in d3b99a7fc7ccb700217b34233a08e3795153422c